### PR TITLE
Rework article choice logic

### PR DESCRIPTION
**Pull request by brunobesson**
Don't know why, but I apparently made mistake when copying stuff from v5

### CODE REVIEW

File: c2corg_ui/templates/utils/__init__.py [hunk 1]
```diff
@@ -95,18 +95,38 @@ def get_route_gear_articles(route):
...
+        if 'rock_climbing' in activities:
+            if route.get('equipment_rating') in ['P1', 'P1+']:
+                articles.append(('183332', 'bolted rock climbing gear'))
+            elif 'mountain_climbing' not in activities and \
+                    route.get('global_rating') in [
+                        'F',
+                        'F+',
+                        'PD',
+                        'PD+',
+                        'AD'
```
Comment (asaunier) [inline c2corg_ui/templates/utils/__init__.py]: What about PD- and AD-?

Comment (brunobesson) [inline c2corg_ui/templates/utils/__init__.py]: You're right. :sob:

**Comment by fbunoz** [comment]
Instead of enumerating all items in a range, is it possible to define a range by > and < conditions ?
Because if we add some subdivision later, the enumeration will be incorrect, and will need an update.

**Comment by brunobesson** [comment]
Feel free to propose something.
AFAIK there is no easy mapping between the possible values and numbers. Maybe in v6_common?

**Comment by asaunier** [comment]
>  there is no easy mapping between the possible values and numbers.

Indeed. The attributes in DB are not numeric but are enumerations of the strings defined in https://github.com/c2corg/v6_common/blob/master/c2corg_common/attributes.py

For instance for ``ski_rating``: 
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/route.py#L78
https://github.com/c2corg/v6_api/blob/master/c2corg_api/models/enums.py#L65-L66
https://github.com/c2corg/v6_common/blob/master/c2corg_common/attributes.py#L314-L333

> Instead of enumerating all items in a range, is it possible to define a range by > and < conditions ?

I don't think it can be used in the UI because the UI has no access to the models (in the API).
In the API, especially for the search requests, it is possible to specify a range of ratings.

**Pull request merged**